### PR TITLE
feat: Cart page added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
-cart page related css is you can find cart.css and Html in cart.html.

**screenshots**
img9
img10

### CODE REVIEW

**Comment by kotalwar-krishna** [comment]
For Width I used here % unit

**Review by psychoactiv — APPROVED** [review]

**Comment by psychoactiv** [comment]
looks good to merge

File: Screens/cart.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    width: 10rem;
+    text-align: center;
+}
+
+.cart-container .cart-table td:nth-child(2),
+.cart-container .cart-table td:nth-child(4),
+.cart-container .cart-table td:nth-child(5),
+.cart-container .cart-table td:nth-child(6){
+    width: 14rem;
+    text-align: center;
```
Comment (Namrata-J) [inline Screens/cart.css:50]: as you are using text-align repeatedly, you can add it your utility classes(just a suggestion)

**Review by Namrata-J — APPROVED** [review]

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    <title>sarathi-cart</title>
+    
+    <!-- font awesome cdn link  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    
+    <!--  css file link  -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- My Component Library link -->
+    <link rel="stylesheet" href="https://sarathi-ui.netlify.app/components/component.css">
```
Comment (sakleprajwal) [inline cart.html:15]: If you are using the first and last link in all HTML pages, I will suggest including it in your base.css file.
You can use the below code in base.css:
@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css')
@import url('https://sarathi-ui.netlify.app/components/component.css')

File: style.css [hunk 3]
```diff
@@ -259,17 +262,35 @@ section{
...
+   }
+   .cart-checkout .apply-coupan {
+    width: 100%;
+  }
+  .cart-checkout .cart-total {
+    width: 100%;
+  }
 }
 
 @media (max-width:450px){
```
Comment (sakleprajwal) [inline style.css:277]: Ideal max-width for mobile screen is 576 as per my knowledge. You can do your own research.

**Review by sakleprajwal — APPROVED** [review]
Suggested some changes. Everything else looks good.

File: cart.html (hunk 2 above)
Comment (kotalwar-krishna) [reply cart.html:15]: Ok I will do that

File: style.css (hunk 3 above)
Comment (kotalwar-krishna) [reply style.css:277]: ok, I dont know , I will do this change

File: Screens/cart.css (hunk 1 above)
Comment (kotalwar-krishna) [reply Screens/cart.css:50]: ok

**Pull request merged**